### DISCUSSION TITLE
Rancher Desktop 1.6.2

### DISCUSSION THREAD

**Discussion by jandubois** [rancher-sandbox/rancher-desktop#3341] [General]
This is the 1.6.2 release of Rancher Desktop, an open source desktop application to bring Kubernetes and container management to Mac, Windows, and Linux.

## This patch release fixes an incompatibility with macOS Ventura

Rancher Desktop 1.6.0 and 1.6.1 do not work properly with macOS 13 Ventura. When running Kubernetes versions 1.24.* or 1.25.*, the VM would hang right away before becoming ready. With earlier versions, Kubernetes would work initially, but start hanging after restarting Rancher Desktop. This issue has been fixed.

There are no other changes in this release compared to 1.6.1.

## Connect with the developers

- The [issue queue](https://github.com/rancher-sandbox/rancher-desktop/issues)

- [Rancher Users Slack](https://slack.rancher.io/) in the #rancher-desktop channel

## Changelog

The [full version changelog, from v1.6.1, can be found using GitHub compare](https://github.com/rancher-sandbox/rancher-desktop/compare/v1.6.1...v1.6.2) and the details of the release can be found in the [v1.6.2 milestone](https://github.com/rancher-sandbox/rancher-desktop/milestone/27?closed=1).

<hr /><em>This discussion was created from the release <a href='https://github.com/rancher-sandbox/rancher-desktop/releases/tag/'>Rancher Desktop 1.6.2</a>.</em>

**Comment by jimtyrro** [comment]
Hi,

Looking for help on Rancher Desktop install for MacOS 12.6.1.
I’ve downloaded the latest version 1.6.2 dmg and installed it but when trying to install nerdctl I don’t see any option to activate/install it in the dashboard or Preferences. In the docs its says:
After Rancher Desktop is installed, users will have access to these supporting utilities:
Helm
kubectl
nerdctl
docker (moby)
So where do I find these utilities to activate/install or do they supposed to be installed automatically?

I've already installed helm and kubectl separately via brew, will this be a problem?

When trying to running nerdctl it says command not found. Is there something I've missed about proper installation procedure?

Please steer me in the right direction.

Thank you.

**Comment by jandubois** [comment]
You need to put `~/.rd/bin` on your `PATH`. The app asked you during first-run if you wanted to do it manually, of if it should do it for you automatically. I assume you selected "Manually" if the tools are not on your `PATH` (or that you have not opened a new shell since starting Rancher Desktop the first time).

If you let Rancher Desktop add the directory to the `PATH`, then it will prepend it to the front. So it will shadow your versions of `helm` and `kubectl`. If you want to continue to use the brew versions, then you should add the directory manually to the `PATH` at the end.

One thing to note though is that we use [kuberlr](https://github.com/flavio/kuberlr) to manage `kubectl`, so you will always use a compatible version, regardless which Kubernetes version you switch to.

**Comment by jimtyrro** [comment]
Hi Jan,

Thank you for replying and explaining the fix. Actually install did add
this path but I just forgot to restart the terminal for it to take effect.
It’s all good now.

Thanks again.


On November 11, 2022 at 02:42:24, Jan Dubois ***@***.***)
wrote:

You need to put ~/.rd/bin on your PATH. The app asked you during first-run
if you wanted to do it manually, of if it should do it for you
automatically. I assume you selected "Manually" if the tools are not on
your PATH (or that you have not opened a new shell since starting Rancher
Desktop the first time).

If you let Rancher Desktop add the directory to the PATH, then it will
prepend it to the front. So it will shadow your versions of helm and kubectl.
If you want to continue to use the brew versions, then you should add the
directory manually to the PATH at the end.

One thing to note though is that we use kuberlr
<https://github.com/flavio/kuberlr> to manage kubectl, so you will always
use a compatible version, regardless which Kubernetes version you switch to.

—
Reply to this email directly, view it on GitHub
<https://github.com/rancher-sandbox/rancher-desktop/discussions/3341#discussioncomment-4111516>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ADCIAPVLHICZTDGCAEYFOBTWHVQLRANCNFSM6AAAAAARWUQ34M>
.
You are receiving this because you commented.Message ID:
<rancher-sandbox/rancher-desktop/repo-discussions/3341/comments/4111516@
github.com>

**Comment by mingjiana** [comment]
Prompt for Injector.avv virus when downloading
